app/services/user_validation_service.rb

```diff
@@ -49,7 +49,7 @@ def validate_user(user, task_id = nil, request = nil)
     return ValidateResult.new(
       :fail,
       "Logins not allowed, no providers are being managed yet. Please contact the administrator"
-    ) unless db_user.super_admin_user? || data_ready?
+    ) unless db_user.super_admin_user?
 
```

Comment:
@kbrock So wait, this will now fail the login unless the user is super_admin (in other words, for all users BUT super_admin)?  I think we want to remove this entire check so it never fails on `data_ready?` being false.